README.md

```diff
@@ -1454,6 +1524,211 @@ client
   })
 ```
 
+## Version actions
+### Create version
+Create a draft or release version of a published document.
+```js
+client.action(
```

Comment:
Maybe its already prevalent in existing code examples here, but seems like theres a bit of inconsistent usage of `try {await …} catch {…}` and `.then(…).catch(…)` here. Probably not worth addressing right now though.